src/lib/public-mappings/manufacturer.ts

```diff
@@ -0,0 +1,34 @@
+import type { Manufacturer } from "@seamapi/types/devicedb"
+
+import type { StoredManufacturer } from "lib/models.ts"
+
+import { publicMapImageReference } from "./image-reference.ts"
+
+interface PublicMapManufacturerOptions {
+  fake_devicedb_endpoint: string
+  manufacturer: StoredManufacturer
+  live_seam_connect_endpoint?: string
+  x_forwarded_seam_base_url?: string
```

Comment:
I am finding keeping track of `fake_devicedb_endpoint` / `live_seam_connect_endpoint`/ and `x_forwarded_seam_base_url` at each level is very confusing. I do not understand what they all mean. Are all three of these values only used to resolve the image url proxy? If so, maybe we can just add a middleware that adds `req.image_proxy_url` to the request and that can be passed around instead.

Reply:
yeah, I think we can coalesce `fake_devicedb_endpoint` and `x_forwarded_seam_base_url` into one now

Reply:
0d7a925